tests/test_definitions/bigquery_test/expect_column_values_to_be_unique.json

```diff
@@ -1,50 +0,0 @@
-{
```

Comment:
@Shinnnyshinshin I think it would be good to use the existing test definitions, e.g. tests/test_definitions/column_map_expectations/expect_column_values_to_be_unique.json. This PR changes it so these existing files are used instead.